### ISSUE TITLE
Headings, HTML structure

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4098]
**Is your feature request related to a problem? Please describe.**
VIVO should be aligned with AA level of WCAG regarding **headings and HTML structure**.
